app/src/main/kotlin/com/intive/picover/images/view/ImagesScreen.kt

```diff
@@ -0,0 +1,77 @@
+package com.intive.picover.images.view
+
+import android.net.Uri
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import com.intive.picover.common.loader.PicoverLoader
+import com.intive.picover.images.viewmodel.FirebaseStorageState
+import com.intive.picover.images.viewmodel.FirebaseStorageState.Error
+import com.intive.picover.images.viewmodel.FirebaseStorageState.Loading
+import com.intive.picover.images.viewmodel.FirebaseStorageState.Success
+import com.intive.picover.images.viewmodel.ImagesViewModel
+import com.skydoves.landscapist.coil.CoilImage
+
+@Composable
+fun ImagesScreen(viewModel: ImagesViewModel) {
+	val state by viewModel.uiState.collectAsState()
+	ValidateFirebaseStorageState(state)
+}
+
+@Composable
+private fun ValidateFirebaseStorageState(state: FirebaseStorageState) {
+	when (state) {
+		is Success -> {
+			val uris = state.uris
+			LazyColumn(
+				content = {
+					items(uris) {
+						ListItem(uri = it)
+					}
+				},
+				horizontalAlignment = Alignment.CenterHorizontally,
+				verticalArrangement = Arrangement.Center,
+			)
+		}
+		is Error -> {
+			// TODO-KMA Handle the error
```

Comment:
we could extract this work to an issue of creating PicoverError :) and do it later